### PR TITLE
STORM-3061: thrift 0.11

### PR DESCRIPTION
**Pull request by revans2**
This moves to thrift 0.11

The only files changed are pom.xml and genthrift.sh.  The other files are all generated.

### CODE REVIEW

File: storm-client/src/genthrift.sh [hunk 1]
```diff
@@ -17,7 +17,7 @@
 
 rm -rf gen-javabean gen-py py
 rm -rf jvm/org/apache/storm/generated
-thrift --gen java:beans,hashcode,nocamel,generated_annotations=undated --gen py:utf8strings storm.thrift
```
Comment (Ethanlm) [inline storm-client/src/genthrift.sh]: Could you explain the reason to remove `hashcode` and  the impact of it? I am not familiar with it. Thanks

Comment (revans2) [reply storm-client/src/genthrift.sh]: Not sure why we had hashcode to begin with. From the 0.9.3 compiler --help it shows
```
  csharp (C#):
...
    hashcode:        Generate a hashcode and equals implementation for classes.
```

So it is generating hash codes for the c# language.  I believe that it was also being used for python, but that is not documented in thrift.  In 0.11.0 it is still there in the help command for c#, but now if I run with out the change I get the error.

```
[FAILURE:generation:1] Error: unknown option java:hashcode
```

Comment (srdo) [reply storm-client/src/genthrift.sh]: There used to be a hashcode option for Java, but it was removed https://github.com/apache/thrift/pull/65. As of Thrift 0.10, it now complains if you try to use thrift with invalid arguments https://issues.apache.org/jira/browse/THRIFT-3605

Comment (srdo) [reply storm-client/src/genthrift.sh]: I think the utf8strings option may be unnecessary https://issues.apache.org/jira/browse/THRIFT-3503

Comment (revans2) [reply storm-client/src/genthrift.sh]: @srdo do you want me to remove the utf8strings option?

Comment (srdo) [reply storm-client/src/genthrift.sh]: No, it's fine

**Comment by srdo** [comment]
+1

**Comment by arunmahadevan** [comment]
+1 LGTM.

**Comment by revans2** [comment]
The integration test failures look unrelated to this change.

**Comment by revans2** [comment]
Yes I reran the integration tests manually and they all passed.

**Pull request merged**